perses/annihilation/lambda_protocol.py

```diff
@@ -102,6 +102,12 @@ def __init__(self, functions='default'):
                                   lambda x: x,
                                   'lambda_torsions':
                                   lambda x: x}
+            elif self.type == 'ele-scaled':
```

Comment:
Maybe add a comment here that this is "linear scaling of electrostatic energy for insertion and deletion"?

Reply:
Done in docstrings